doc/nvim-tree-lua.txt

```diff
@@ -539,6 +539,32 @@ This will be experimental for a few weeks and will become the default.
     Idle milliseconds between filesystem change and action.
       Type: `number`, Default: `50` (ms)
 
+*nvim-tree.on_attach*
```

Comment:
This should be more obvious, something like `nvim-tree.apply_mappings` or `setup_mappings`.

Reply:
I believe on_attach is good enough though, it respects the neovim api ecosystem.
Also this could be used for other things, such as setting buffer options or window options.

Reply:
> setting buffer options or window options

That works.